test/integration/error-node-builtin-unresolved/expected-output.txt

```diff
@@ -1,5 +1,4 @@
 ⠼ snowpack installing... bad-node-builtin-pkg, http
 ✖ ../../../node_modules/bad-node-builtin-pkg/entrypoint.js
    "XXXX" (Node.js built-in) could not be resolved. (https://www.snowpack.dev/#node-built-in-could-not-be-resolved)
-✖ "XXXX" (Node.js built-in) could not be resolved. (https://www.snowpack.dev/#node-built-in-could-not-be-resolved). See https://www.snowpack.dev/#troubleshooting
```

Comment:
Just a note: it seems like pulling `paint()` out of build caused this to be flakier. In many cases we’re logging twice: once in the `rollup-plugin-catch-unresolved`, then again in `install#L455`. I’m not sure what causes it, or if it’s an error with Rollup, our plugin, or us.

Reply:
Ah actually this seems to be directly related to `ora` and Windows. That seems to blow away existing messages in cases.

Reply:
can't wait to remove ora :)